back-end/API/app/services/user_service.py

```
@@ -1,144 +1,90 @@
+        user.lastName = sanitize_html(dto.lastName)
+
+        return user_repo.update(user)
+
+    def toggle_admin(self, current_user, id) -> UserReadDTO:
+        user = user_repo.find_by_id(id)
+        if current_user.id == user.id:
+            raise PermissionException("Un administrateur ne peut pas changer son propre statut d'administrateur")
+        user.isModerator = not user.isModerator
+        return user_repo.update(user)
+
+    def delete(self, current_user, id: int) -> UserReadDTO:
```

Ruff (0.15.13)

[error] Function argument `id` is shadowing a Python builtin

(A002)